src/main/java/ledger/user_interface/ui_controllers/component/UserTransactionInput.java

```diff
@@ -40,6 +40,10 @@
     @FXML
     private TextField tagText;
     @FXML
+    private AccountDropdown account2Text;
```

Comment:
Lets rename these to be meaningfull. sourceAccount and destinationAccount seem good